gameList.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+$db = new PDO ('mysql:host=db;dbname=collection', 'root', 'password');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$testIfEmpty = $_POST;
+$testIfEmpty[] = "";
+if(isset($_POST) && count(array_unique($testIfEmpty)) !== 1 ){
+    addStats($_POST['name'],$_POST['genre'],$_POST['length'], $_POST['price'],$db);
+}
+
+$fullQuery = $db->prepare( 'SELECT `name`, `genre`, `length`, `price` FROM `games`;');
+
+$fullQuery->execute();
+
+$data = $fullQuery -> fetchAll();
```

Comment:
Same deal here, a little function for grabbing everything out of the collection would be a good idea!